libmamba/src/core/transaction.cpp

```diff
@@ -1468,7 +1473,7 @@ namespace mamba
                 }
                 else
                 {
-                    if (!need_pkg_download(s, m_multi_cache))
+                    if (!need_pkg_download(mk_pkginfo(m_pool, s), m_multi_cache))
```

Comment:
`mk_pkginfo` returns a `PackageInfo` right? It replaces a solvable here?

Reply:
Yes, the previous line was calling an implicit conversion from `Solvable*` to `PackageInfo`.